### DISCUSSION TITLE
Taro如何打包成app？

### DISCUSSION THREAD

**Discussion by 1164226245** [NervJS/taro#12986] [Q & A] [answered]
本地开发完Taro项目。想发布到H5和app端，请问是使用yarn build:h5编译成H5发布，然后使用yarn build:rn --platform ios编译成app发布吗

**Comment by zhiqingchen** [comment, marked as answer]
https://docs.taro.zone/docs/react-native

**Reply by rocona** [reply]
请问该教程在Xcode15.4版本还能运行么？已经弄了三天了，现在编译成功，模拟器启动，但是报错了，
`ExpoModulesCore/ViewModuleWrapper.swift:41: Fatal error: Something unexpected has happened. Only dynamically created `ViewModuleWrapper` can be initialized without params.`
